homeassistant/components/fully_kiosk/sensor.py

```diff
@@ -139,3 +151,26 @@ def native_value(self) -> StateType:
             return self.entity_description.state_fn(value)
 
         return value  # type: ignore[no-any-return]
+
+
+class FullyUrlSensor(FullySensor):
+    """Representation of a Fully Kiosk Browser URL sensor."""
```

Comment:
Instead of creating new sensor classes, it is preferred to extend the entity description (e.g., by providing an lambda in it).